docs/BuildFromSource.md

```diff
@@ -278,9 +278,21 @@ On macOS/Linux:
 
 ### Building a subset of the code
 
-This repository is large. Look for `build.cmd`/`.sh` scripts in subfolders. These scripts can be used to invoke build and test on a smaller set of projects.
+When working in the repository, you'll typically be focused on one project area, such as Blazor and SignalR. In that case, it's more epedient to use the `build.cmd` and `build.sh` that are available in each subfolder. When invoked in a subfolder on Windows:
```

Comment:
epedient ?

Reply:
Should've been expedient. Autocorrect, why y do dis?!!?